enterprise/backend/src/metabase_enterprise/metabot_v3/tools/interface.clj

```diff
@@ -47,6 +47,10 @@
 
 (mr/def ::metadata.name
   [:and
+   {:encode/api-request   (fn [x]
+                            (u/->snake_case_en (name x)))
+    :decode/api-response  (fn [x]
+                            (keyword "metabot.tool" (name (u/->kebab-case-en x))))}
```

Comment:
will this break if we move the tools to a different namespace?

Reply:
no. the namespace of the keywords here (`:metabot.tool/confirm-invite-user`) are not related to the namespaces that the tools are defined in. That function is in `metabase-enterprise.metabot-v3.tools.confirm-invite-user` and the keyword's namespace